modules/conflict_resolver/help/conflict_resolver.md

```diff
@@ -1,9 +1,29 @@
 # Conflict Resolver
 
-This module is designed to identify and resolve data conflicts in a study. LORIS uses double data entry, which means you may be asked to enter critical data twice in duplicate instrument forms. The Conflict Resolver will flag any discrepancies between the two entries (called *unresolved conflicts*) and allow you to fix or "resolve" them easily. 
+The Conflict Resolver module enables studies to identify and resolve
+typos and discrepancies that arise between initial data entry and double
+data entry forms.
 
-Conflicts are displayed in a table with two separate tabs—resolved and unresolved conflicts. By default, the module displays the *Unresolved Conflicts* tab. You can narrow this list of results using any combination of filters in the *Selection Filters* section. 
+Unresolved and Resolved conflicts are displayed on two separate tabs of
+the module. Unresolved conflicts are displayed by default upon accessing
+the module. To access resolved conflicts, click on the “Resolved” tab.
 
-You can resolve all conflicts in the **Correct Answer** column of the resulting table. Click the drop-down to reveal two different responses—this is the conflict. These are two distinct values that were entered initially and during double data entry—they should be the same value. After cross-checking your data records for accuracy, select the correct response of the two. 
+Selection filters can be applied to search for a particular subject
+and/or instrument. Click the “Clear Form” button to reset all
+Selection Filters.
 
-You can continue resolving conflicts in this list. To save your changes, click **Save** below the table. All saved, resolved conflicts will now appear in the *Resolved Conflicts* tab of the table. 
+By default, unresolved data entry conflicts appear as “Unresolved”
+under the “Correct Answer” column, until a user manually resolves
+the data discrepancy. To resolve a conflict, click in the “Correct
+Answer” column dropdown select for a given data entry item. This
+dropdown-select will show two options: these are the data values that were
+entered in each of the initial and the double data entry forms. Select
+one of these options as the correct answer, after cross-checking against
+original data records (e.g. paper forms) for accuracy. Changes are saved
+automatically. Once a conflict is resolved a green check will appear
+beside the input filed. If a red cross is displayed, the conflict could
+not be saved (Your browser console log will display the error message.)
+
+It is possible to change the resolved conflicts to a new value until
+the page is refreshed. After refresh or when changing tab, the newly
+resolved conflicts will appear in the “Resolved” tab.
```

Comment:
@ridz1208 Not sure if I should be keeping the old help text here.

Reply:
This is an enhanced behaviour where each conflict is resolved when a value is selected instead of when the (RIP) save button is hit. Also, Conflicts can be re-resolved by selecting a new value in the drop down.